api/tests/unit/application/tags/index_test.js

```diff
@@ -0,0 +1,36 @@
+const {
+  domainBuilder,
+  expect,
+  sinon,
+  HttpTestServer,
+} = require('../../../test-helper');
+
+const moduleUnderTest = require('../../../../lib/application/tags');
+
+const tagController = require('../../../../lib/application/tags/tag-controller');
+
+describe('Unit | Application | Router | tag-router', () => {
+
+  it('return a response with an http status code OK', async () => {
+    // given
+    const tag1 = domainBuilder.buildTag({ name: 'TAG1' });
+    const tag2 = domainBuilder.buildTag({ name: 'TAG2' });
+    const tag3 = domainBuilder.buildTag({ name: 'TAG3' });
+    const allOrganizationsTags = [tag1, tag2, tag3];
+
+    const method = 'GET';
+    const url = '/api/tags';
+
```

Comment:
Toujours suite à la remarque sur l'accès à la route par un utilisateur non PixMaster, l'ajout du pre-handler rend ce test invalide.

Il faudrait rajouter la ligne suivante 

```suggestion
const securityPreHandlers = require('../../../../lib/application/security-pre-handlers');

```

Reply:
Done